config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -235,13 +235,13 @@ presubmits:
           - verify
         env:
         - name: GOMAXPROCS
-          value: "2"
+          value: "4"
         resources:
           requests:
-            cpu: "2"
+            cpu: "4"
             memory: "6Gi"
```

Comment:
you should probably allocate comparable memory?

1.5gi/core is pretty low, I'd aim for 2gi/core to start.

---

cc @mahamed @xmudrii interesting optimization problem: some jobs may be going slower due to artificially memory limiting themselves while still creating a cluster scaling bottleneck around allocated cores.

we should consider enforcing a core:mem limit to simplify this and make expectations more uniform between the EKS/GKE build clusters

Reply:
recurring topic https://github.com/kubernetes/test-infra/pull/32315#discussion_r1546983694

Reply:
Please set the memory to 8Gi, as recommended

Reply:
Done.